src/main/java/mate/controller/car/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,28 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.service.CarService;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private CarService carService =
+                (CarService) injector.getInstance(CarService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        HttpSession httpSession = req.getSession();
+        Long id = (Long) httpSession.getAttribute("driverId");
+        List<Car> allCarsForCurrentDriver = carService.getAllByDriver(id);
+        req.setAttribute("drivers", allCarsForCurrentDriver);
```

Comment:
```suggestion
        req.setAttribute("cars", allCarsForCurrentDriver);
```
Можливо так буде краще?

Reply:
Agree